conda_env/cli/main.py

```diff
@@ -45,6 +45,13 @@ def do_call(args, parser):
     # func_name should always be 'execute'
     from importlib import import_module
 
+    # Run the pre_command actions
+    command = relative_mod.replace(".main_", "")
+    actions = context.plugin_manager.get_pre_command_hook_actions(
+        f"env_{command}", args
+    )
+    tuple(action() for action in actions)
```

Comment:
```suggestion
    _run_pre_commands(f"env_{command}", args)
```
But this also needs an import above for `conda.cli.conda_argparse._run_pre_commands`